app/uk/gov/hmrc/softdrinksindustrylevy/connectors/ContactFrontendConnector.scala

```diff
@@ -21,21 +21,15 @@ import java.time.{Instant, LocalDateTime, ZoneId}
 import play.api.Configuration
 import play.api.Mode
 import play.api.libs.json.Json
-import uk.gov.hmrc.http.{HeaderCarrier, HttpResponse}
-import uk.gov.hmrc.play.bootstrap.config.RunMode
-import uk.gov.hmrc.play.bootstrap.http.HttpClient
+import uk.gov.hmrc.http.{HeaderCarrier, HttpClient, HttpResponse}
 import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
 import uk.gov.hmrc.softdrinksindustrylevy.models.Subscription
 import uk.gov.hmrc.softdrinksindustrylevy.models.json.internal.subscriptionFormat
 
 import scala.concurrent.{ExecutionContext, Future}
 
-class ContactFrontendConnector(
-  http: HttpClient,
-  val mode: Mode,
-  val runModeConfiguration: Configuration,
-  val runMode: RunMode)
-    extends ServicesConfig(runModeConfiguration, runMode) {
+class ContactFrontendConnector(http: HttpClient, val mode: Mode, val runModeConfiguration: Configuration)
```

Comment:
This no longer needs to be called runModeConfiguration, configuration would be better